### DISCUSSION TITLE
ffmpegHelpers Error

### DISCUSSION THREAD

**Discussion by LordSith77** [advplyr/audiobookshelf#501] [Q&A]
When trying to scan for new books I just added, I keep getting this error.  What am I missing here?

`2022-04-24 21:32:13

ERROR

[FfmpegHelpers] Extract Cover Error Error: ffmpeg exited with code 1: av_interleaved_write_frame(): Input/output error frame= 1 fps=0.0 q=6.9 size=N/A time=00:00:00.00 bitrate=N/A speed= 11x frame= 1 fps=0.0 q=6.9 Lsize=N/A time=00:00:00.00 bitrate=N/A speed=0.00149x video:30kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: unknown Conversion failed!`

**Comment by advplyr** [comment]
That error shouldn't cause any crash or anything. Whatever the embedded cover art is in that audio file it wasn't able to get extracted from the audio file. I'm guessing you also posted this in the unraid forum.

**Reply by LordSith77** [reply]
I did post there as well. Wasn't sure which one was the best place. Sorry about that.

I do have a few audiobooks, that are a single file and over 1GB in size. I'm wondering if those are what causing the error?

I'm going to try and remove those large files, and try scanning again, see if those are what's stopping the other new books from showing up.
